types/k6/test/timers.ts

```diff
@@ -33,7 +33,16 @@ setInterval('functionRef', 5);
 // @ts-expect-error
 setInterval('functionRef', 5, 'arg');
 
+// @ts-expect-no-error
+const intervalId = setInterval(function(){}, 5);
+
 // clearInterval
 
+// @ts-expect-no-error
```

Comment:
```suggestion
```

same as in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63961/files#r1112686419